.github/workflows/ci.yml

```
@@ -5,7 +5,8 @@ on:
     branches: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)